<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bocchi the Paketto!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="rawPacketTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>441</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="xorCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR Key :</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="xorKeyTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="packetInfoGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>441</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Packet Info</string>
    </property>
    <widget class="QLabel" name="magicStartLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Magic Start :</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="magicStartTextEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>4567</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="cmdIdHexTextEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QLabel" name="cmdIdLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CmdId :</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="cmdIdDecTextEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>DEC</string>
     </property>
    </widget>
    <widget class="QLabel" name="cmdIdDirectLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>————&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="metadataLengthHexTextEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QLabel" name="metadataLengthLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Metadata Length :</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="metadataLengthDecTextEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>DEC</string>
     </property>
    </widget>
    <widget class="QLabel" name="metadataLengthDirectLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>————&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="bodyLengthDecTextEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>DEC</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="bodyLengthHexTextEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QLabel" name="bodyLengthDirectLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>————&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="bodyLengthLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Body Length :</string>
     </property>
    </widget>
    <widget class="QLabel" name="metadataLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Metadata :</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="metadataTextEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="bodyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Body :</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="bodyTextEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>281</width>
       <height>151</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="magicEndTextEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>390</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>89ab</string>
     </property>
    </widget>
    <widget class="QLabel" name="magicEndLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Magic End :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_O"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <action name="actiong">
   <property name="text">
    <string>g</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开(O)...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
